src/client/app/heroes/heroes/heroes.component.ts

```diff
@@ -2,8 +2,10 @@ import { Component, OnInit, ChangeDetectionStrategy, OnDestroy } from '@angular/
 import { FormControl } from '@angular/forms';
 
 import { AppSelectors } from '../../store/app-config';
-import { EntityAction, EntityActions, EntityDispatcher, EntitySelectors$,
-  EntityService, OP_ERROR, OP_SUCCESS } from '../../../ngrx-data';
+import {
+  EntityAction, EntityActions, EntityDispatcher, EntitySelectors$,
```

Comment:
yay! now we have even more stuff to import! (boo)

When we started, we both disliked all the crazy stuff we had to import and inject. That's where we came up with a facade. 

Again, no action needed, just restating one of our early distastes. We can discuss to see what we can do.

Reply:
Good point. I forgot to remove some stuff. But maybe we need to delete more.